src/StructuringElements/utils.jl

```diff
@@ -0,0 +1,94 @@
+"""
+    upper, lower = strel_split([T], se)
+
+Split a structuring element into its upper and lower half parts.
+
+The splited non-symmetric SE parts will be represented as array of `T`, where `T` is either
+a `Bool` or `CartesianIndex`. By default, `T = eltype(se)`.
+
+```jldoctest strel_split; setup=:(using ImageMorphology; using ImageMorphology.StructuringElements)
+julia> se = strel_diamond((3, 3))
+3×3 SEDiamondArray{2, 2, UnitRange{$Int}, 0} with indices -1:1×-1:1:
+ 0  1  0
+ 1  1  1
+ 0  1  0
+
+julia> upper, lower = strel_split(se);
+
+julia> upper
+3×3 OffsetArray(::Matrix{Bool}, -1:1, -1:1) with eltype Bool with indices -1:1×-1:1:
+ 0  1  0
+ 1  1  0
+ 0  0  0
+
+julia> lower
+3×3 OffsetArray(::Matrix{Bool}, -1:1, -1:1) with eltype Bool with indices -1:1×-1:1:
+ 0  0  0
+ 0  1  1
+ 0  1  0
+```
```

Comment:
Warning if it remind it properly Erode(Erode(Lower),upper)!=Erode(diamond) the composition lower+upper is not equivalent
Todo check this with tests :)